src/main/webapp/app/exercises/programming/shared/instructions-render/extensions/programming-exercise-plant-uml.extension.ts

```diff
@@ -45,6 +45,9 @@ export class ProgrammingExercisePlantUmlExtensionWrapper implements ArtemisShowd
                 tap((plantUmlSvg: string) => {
                     const plantUmlHtmlContainer = document.getElementById(`plantUml-${index}`);
                     if (plantUmlHtmlContainer) {
+                        // We add a max-width: 100% style attribute to the svg element
+                        // to prevent overflowing the enclosing container
+                        plantUmlSvg = plantUmlSvg.replace(`style="width`, `style="max-width:100%;width`);
```

Comment:
@FrankeLukas hope this is an allowed fixed for the overflow problem on an exercise page with rendered uml model (as svg). It was not possibly for me to solve the problem without restricting the svg width to max of 100%.